.github/workflows/build.yaml

```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   rustfmt:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Updated to ubuntu 22.04 since it's now supported by Intel install packages.

Reply:
I'm fine with this to be `22.04`.